core/Updates/3.7.0-b1.php

```diff
@@ -29,8 +29,8 @@ public function __construct(MigrationFactory $factory)
     public function getMigrations(Updater $updater)
     {
         return [
-            $this->migration->db->addColumn(Model::$rawPrefix, 'evolution_graph_within_period', 'TINYINT(4) NOT NULL DEFAULT 0'),
-            $this->migration->db->addColumn(Model::$rawPrefix, 'evolution_graph_period_n', 'INT(11) NULL'),
+            $this->migration->db->addColumn('report', 'evolution_graph_within_period', 'TINYINT(4) NOT NULL DEFAULT 0'),
```

Comment:
When not using `Model::$rawPrefix` the `use` statement for the model could be removed